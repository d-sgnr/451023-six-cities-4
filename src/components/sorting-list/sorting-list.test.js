import React from "react";
import renderer from "react-test-renderer";
import SortingList from "./sorting-list.jsx";

it(`SortingList should be rendered correctly`, () => {
  const tree = renderer.create(
      <SortingList
        activeSortType={`POPULAR`}
        onSortingChange={() => {}}
      />).toJSON();

  expect(tree).toMatchSnapshot();
});
