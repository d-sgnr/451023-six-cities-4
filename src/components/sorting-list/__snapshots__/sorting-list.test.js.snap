// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortingList should be rendered correctly 1`] = `
Array [
  <ul
    className="places__options places__options--custom places__options--opened"
  >
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex="0"
    >
      Popular
    </li>
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex="0"
    >
      Price: low to high
    </li>
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex="0"
    >
      Price: high to low
    </li>
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex="0"
    >
      Top rated first
    </li>
  </ul>,
  <select
    className="places__sorting-type visually-hidden"
    id="places-sorting"
    onChange={[Function]}
    value=""
  >
    <option
      className="places__option"
      value="popular"
    >
      Popular
    </option>
    <option
      className="places__option"
      value="to-high"
    >
      Price: low to high
    </option>
    <option
      className="places__option"
      value="to-low"
    >
      Price: high to low
    </option>
    <option
      className="places__option"
      value="top-rated"
    >
      Top rated first
    </option>
  </select>,
]
`;
